---
permalink: /
title: "About me"
excerpt: "Ecology and Evolution of Arthropods and their symbionts"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am an evolutionary biologist interested in understanding the evolution of gene-gene interactions between symbiont genomes and nuclear genomes in insects. Through evolution, insects have acquired intimate symbiotic microorganisms, including microbes and mitochondria, which provided great benefits to insect hosts, facilitating the adaptation and diversification of the hosts. Symbiont genes (including both microbial and mitochondrial genes) and host nuclear genes cooperate closely with each other in critical pathways, such as energy production and essential amino acids biosynthesis. However, as symbiont genomes are haploid, maternally inherited, and lacking in recombination, symbiont genomes exhibit higher evolutionary rates than host nuclear genes. This incompatibility between symbiont and host genes could lead to inviable offspring and change of sex ratio, which can lead to change of host fitness and speciation events. 

Using next generation sequencing and bioinformatic tools, I am interested in: 
1. What are the genes involved in host-symbiont interactions?
1. How do arthropod hosts response to the deleterious mutations accumulated in the symbiont genes? 
1. How do symbionts interact with each other within the same host?


Education
======
2018 – present Postdoctoral Researcher 
               University of Texas, Austin, USA					                                          
               Advisor: Nancy A. Moran
              
2013 – 2018		 Ph.D. in Biological Sciences                                                                                                                                      University of Notre Dame, Notre Dame, IN, USA
               Advisor: Michael E. Pfrender

2007 – 2011	   Bachelor of Engineering in Biomedical Engineering
          		 Southeast University, Nanjing, China                                                           

Publications
======
19. Li, Y., Zhang B., and Moran, N.A., 2020. The aphid X chromosome is a dangerous place for functionally important genes: diverse evolution of Hemipteran genomes based on chromosome-level assemblies. Molecular Biology and Evolution. [link]

18. Thomas, G. W., (Li, Y. with other 78 co-authors) et al. 2020. Gene content evolution in the arthropods. Genome Biology, 21(1), 1-14. [link]

17. Zhang, B., Leonard, S.P., Li, Y., Moran, N.A., 2019. Obligate bacterial endosymbionts limit thermal tolerance of insect host species. Proceedings of the National Academy of Sciences, 116(49), pp.24712-24718. [link]

16. Ryan, S.F., Lombaert, E., Espeset, A., Vila, R., Talavera, G, Dincă, V., Doellman, M.M., Renshaw, M.A., Eng, M.W., Hornett, E.A., Li, Y., Pfrender, M.E., Shoemaker, D., 2019. Global invasion history of the agricultural pest butterfly Pieris rapae revealed with genomics and citizen science. Proceedings of the National Academy of Sciences, 116(40), pp.20015-20024. [link]

15. Li, Y., Park, H. *, Smith, T.E. and Moran, N.A., 2019. Gene family evolution in the pea aphid based on chromosome-level genome Assembly. Molecular Biology and Evolution, 36(10), pp.2143-2156. [link]

14. Meng, G., Li, Y., Yang, C. and Liu, S., 2019. MitoZ: A toolkit for mitochondrial genome assembly, annotation and visualization. Nucleic Acids Research, 47(11), pp.e63-e63. [link]

13. Li, Y., Molik, D. C., Pfrender M. E., 2019. EPPS, a metabarcoding bioinformatics pipeline using Nextflow. Biodiversity Science, 27(5), pp.567-575. [link]

12. Grey, E.K., Bernatchez, L., Cassey, P., Deiner, K., Deveney, M., Howland, K.L., Lacoursière-Roussel, A., Leong, S.C.Y., Li, Y., Olds, B. and Pfrender, M.E., 2018. Effects of sampling effort on biodiversity patterns estimated from environmental DNA metabarcoding surveys. Scientific reports, 8(1), p.8843. [link]

11. Li Y., Evans, N.T., Renshaw, M.A., Jerde, C.L., Olds, B.P., Shogren, A.J., Deiner, K., Lodge, D.M., Lamberti, G.A. and Pfrender, M.E., 2018. Estimating fish alpha- and beta-diversity along a small stream with environmental DNA metabarcoding. Metabarcoding and Metagenomics, 2, p.e24262. [link]

10. Li, Y., Zhang, R., Liu, S., Donath, A., Peters, R.S., Ware, J., Misof, B., Niehuis, O., Pfrender, M.E. and Zhou, X., 2017. The molecular evolutionary dynamics of oxidative phosphorylation (OXPHOS) genes in Hymenoptera. BMC evolutionary biology, 17(1), p.269. [link]

9. Deiner, K., Renshaw, M.A., Li, Y., Olds, B.P., Lodge, D.M. and Pfrender, M.E., 2017. Long‐range PCR allows sequencing of mitochondrial genomes from environmental DNA. Methods in Ecology and Evolution, 8(12), pp.1888-1898. [link]

8. Evans, N.T., Li, Y., Renshaw, M.A., Olds, B.P., Deiner, K., Turner, C.R., Jerde, C.L., Lodge, D.M., Lamberti, G.A. and Pfrender, M.E., 2017. Fish community assessment with eDNA metabarcoding: effects of sampling design and bioinformatic filtering. Canadian Journal of Fisheries and Aquatic Sciences, 74(9), pp.1362-1374. [link]

7. Olds, B.P., Jerde, C.L., Renshaw, M.A., Li, Y., Evans, N.T., Turner, C.R., Deiner, K., Mahon, A.R., Brueseke, M.A., Shirey, P.D. and Pfrender, M.E., 2016. Estimating species richness using environmental DNA. Ecology and Evolution, 6(12), pp.4214-4226. [link]

6. Evans, N.T., Olds, B.P., Renshaw, M.A., Turner, C.R., Li, Y., Jerde, C.L., Mahon, A.R., Pfrender, M.E., Lamberti, G.A. and Lodge, D.M., 2016. Quantification of mesocosm fish and amphibian species diversity via eDNA metabarcoding. Molecular Ecology Resources, 16(1), pp.29-41. [link]

5. Misof, B., (Li, Y. with other 100 co-authors) et al. 2014. Phylogenomics resolves the timing and pattern of insect evolution. Science, 346(6210), pp.763-767. [link]

4. Tang, M., Tan, M., Meng, G., Yang, S., Su, X., Liu, S., Song, W., Li, Y., Wu, Q., Zhang, A. and Zhou, X., 2014. Multiplex sequencing of pooled mitochondrial genomes—a crucial step toward biodiversity analysis using mito-metagenomics. Nucleic Acids Research, 42(22), pp.e166-e166. [link]

3. Liu, S., Li, Y., Lu, J., Su, X., Tang, M., Zhang, R., Zhou, L., Zhou, C., Yang, Q., Ji, Y. and Douglas, W.Y., 2013. SOAPBarcode: revealing arthropod biodiversity through assembly of Illumina shotgun sequences of PCR amplicons. Methods in Ecology and Evolution, 4(12), pp.1142-1150. [link]

2. Zhou, X., Li, Y., Liu, S., Yang, Q., Su, X., Zhou, L., Tang, M., Fu, R., Li, J. and Huang, Q., 2013. Ultra-deep sequencing enables high-fidelity recovery of biodiversity for bulk arthropod samples without PCR amplification. GigaScience, 2(1), p.4. [link]

1. Arboleya, S., Li, A., Margolles, A., Li, Y., Zhang, D., Xiao, L., Solís, G., Fernández, N., Clara, G. and Gueimonde, M., 2012. Deep 16S rRNA metagenomics and quantitative PCR analyses of the premature infant fecal microbiota. Anaerobe, 18(3), pp.378-380. [link](https://doi.org/10.1016/j.anaerobe.2012.04.013)

Site-wide configuration
------
The main configuration file for the site is in the base directory in [_config.yml](https://github.com/academicpages/academicpages.github.io/blob/master/_config.yml), which defines the content in the sidebars and other site-wide features. You will need to replace the default variables with ones about yourself and your site's github repository. The configuration file for the top menu is in [_data/navigation.yml](https://github.com/academicpages/academicpages.github.io/blob/master/_data/navigation.yml). For example, if you don't have a portfolio or blog posts, you can remove those items from that navigation.yml file to remove them from the header. 

Create content & metadata
------
For site content, there is one markdown file for each type of content, which are stored in directories like _publications, _talks, _posts, _teaching, or _pages. For example, each talk is a markdown file in the [_talks directory](https://github.com/academicpages/academicpages.github.io/tree/master/_talks). At the top of each markdown file is structured data in YAML about the talk, which the theme will parse to do lots of cool stuff. The same structured data about a talk is used to generate the list of talks on the [Talks page](https://academicpages.github.io/talks), each [individual page](https://academicpages.github.io/talks/2012-03-01-talk-1) for specific talks, the talks section for the [CV page](https://academicpages.github.io/cv), and the [map of places you've given a talk](https://academicpages.github.io/talkmap.html) (if you run this [python file](https://github.com/academicpages/academicpages.github.io/blob/master/talkmap.py) or [Jupyter notebook](https://github.com/academicpages/academicpages.github.io/blob/master/talkmap.ipynb), which creates the HTML for the map based on the contents of the _talks directory).

**Markdown generator**

I have also created [a set of Jupyter notebooks](https://github.com/academicpages/academicpages.github.io/tree/master/markdown_generator
) that converts a CSV containing structured data about talks or presentations into individual markdown files that will be properly formatted for the academicpages template. The sample CSVs in that directory are the ones I used to create my own personal website at stuartgeiger.com. My usual workflow is that I keep a spreadsheet of my publications and talks, then run the code in these notebooks to generate the markdown files, then commit and push them to the GitHub repository.

How to edit your site's GitHub repository
------
Many people use a git client to create files on their local computer and then push them to GitHub's servers. If you are not familiar with git, you can directly edit these configuration and markdown files directly in the github.com interface. Navigate to a file (like [this one](https://github.com/academicpages/academicpages.github.io/blob/master/_talks/2012-03-01-talk-1.md) and click the pencil icon in the top right of the content preview (to the right of the "Raw | Blame | History" buttons). You can delete a file by clicking the trashcan icon to the right of the pencil icon. You can also create new files or upload files by navigating to a directory and clicking the "Create new file" or "Upload files" buttons. 

Example: editing a markdown file for a talk
![Editing a markdown file for a talk](/images/editing-talk.png)

For more info
------
More info about configuring academicpages can be found in [the guide](https://academicpages.github.io/markdown/). The [guides for the Minimal Mistakes theme](https://mmistakes.github.io/minimal-mistakes/docs/configuration/) (which this theme was forked from) might also be helpful.
